test/spec/stores/users.test.js

```diff
@@ -15,8 +15,9 @@ describe('UsersStore', () => {
 
   it('should overide users on new fetch', (done) => {
     let count = 0;
-    const handleChange = () => {
-      let {users} = store.getState();
+
+    function handleChange() {
+      const { users } = store.getState();
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'destructuring expression' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).